api/envoy/api/v2/cds.proto

```diff
@@ -381,6 +381,13 @@ message Cluster {
   // Configuration for load balancing subsetting.
   LbSubsetConfig lb_subset_config = 22;
 
+  // Specific configuration for the LeastRequest load balancing policy.
+  message LeastRequestLbConfig {
+    // The number of random healthy hosts from which the host with the fewest active requests will
+    // be chosen. Defaults to 2 so that we perform P2C selection if the field is not set.
```

Comment:
nit: Can you spell out P2C and/or link back to the lb overview docs which talk about this in more detail?